hardware/makefiles/lwc_ghdl.mk

```diff
@@ -40,7 +40,7 @@ GENERICS_OPTS=$(shell $(PYTHON3_BIN) $(LWC_ROOT)/hardware/scripts/config_parser.
 ### GHDL analyze testbench files, elaborate, and run
 .PHONY: sim-ghdl help-ghdl clean-ghdl
 sim-ghdl: $(WORK_LIB)-obj$(VHDL_STD).cf $(SIM_VHDL_FILES) Makefile config-vars
-	$(GHDL_BIN) -a $(GHDL_OPT) $(GHDL_WARNS) $(GHDL_ELAB_OPTS) $(SIM_ONLY_VHDL_FILES)
+	$(GHDL_BIN) -a $(GHDL_OPT) $(GHDL_WARNS) $(GHDL_ELAB_OPTS) $(SIM_VHDL_FILES)
```

Comment:
I think this is wrong. It should stay SIM_ONLY_VHDL_FILES as the other common files should have already been elaborated in ` $(WORK_LIB)-obj$(VHDL_STD).cf`

Reply:
I make this change do to something failing to build. I will check to see if it is fixed now that I am rebased of development

Reply:
Changing this back does not appear to break anything on my in so i will.